.storybook/decorators/withLang.tsx

```diff
@@ -3,9 +3,7 @@ import {StoryFn, StoryContext} from '@storybook/react';
 
 import {Lang} from '../../src/models/locale';
 
-import {configure} from '../../src/configure';
-
-configure({lang: Lang.En});
+import {configure} from '@gravity-ui/uikit';
```

Comment:
Is it not necessary to call this configurator?

Reply:
In this decorator, we configure the language for our stories. We can take it out of here, but we'll have to add it somewhere else.